repos/emacs/shell.nix

```diff
@@ -0,0 +1,13 @@
+{ pkgs ? import <nixpkgs> {} }:
+
+let
+  pythonEnv = pkgs.python3.withPackages(ps: [
+    ps.httpx
+    ps.lxml
+  ]);
+
+in pkgs.mkShell {
+  buildInputs = [
+    pythonEnv
+  ];
+}
```

Comment:
It might be nicer if this file was named something like `updater-shell.nix`.

Obviously that would require explicitly passing it to `nix-shell`, but my proposed change on `update` should make that a non-issue.